packages/react/src/utils/renderComponent.tsx

```diff
@@ -93,8 +92,7 @@ const renderComponent = <P extends {}>(
     saveDebug,
     theme: context.theme || emptyTheme,
     performance: {
-      enableVariablesCaching:
-        typeof enableVariablesCaching === 'boolean' ? enableVariablesCaching : true,
```

Comment:
Everything that we want is just to override `enableStylesCaching` there always to `false`